ahfe_tutorial/python_tutorial.ipynb

```diff
@@ -0,0 +1,842 @@
+{
```

Comment:
<p>It would be good to offer a short overview of what the default settings are and what we are changing here. Nothing too fancy, just a "by default we run N repeats with solvent &amp; vacuum simulation lengths of Y and Z over L lambda windows, to speed things up here we instead do...".</p><p> </p><p>Maybe also worth bringing down the number of repeats to 1 to speed up the notebook?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenFreeEnergy/ExampleNotebooks/pull/100/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Reply:
<p>Added this!</p>